src/common/path_util.cpp

```diff
@@ -95,6 +95,13 @@ static auto UserPaths = [] {
         user_dir =
             std::filesystem::path(getenv("HOME")) / "Library" / "Application Support" / "shadPS4";
     }
+#elif defined(__linux__)
+    auto user_dir = std::filesystem::current_path() / PORTABLE_DIR;
+    // Check if the "user" directory exists in the current path:
+    if (!std::filesystem::exists(user_dir)) {
+        // If it doesn't exist, use the new hardcoded path:
+        user_dir = std::filesystem::path(getenv("HOME")) / ".local" / "share" / "shadPS4";
```

Comment:
You're supposed to use `$XDG_DATA_HOME` first before fallbacking to `$HOME/.local/share`.

Reply:
Fixed